requirements.txt

```diff
@@ -1,5 +0,0 @@
-musicbrainzngs
-mutagen
-pycdio>0.20
-PyGObject
-requests
```

Comment:
It might be an idea to replace the with a single line `-e .`? See https://caremad.io/posts/2013/07/setup-vs-requirement/ for some thoughts.

Reply:
That would be good for not breaking compatibility with existing users. I do not like the idea of keeping a requirements.txt that is not used for doing a deployment with a particular version of each dependency.

In the article "application" refers to a web-app, where the environment can be controlled and  the versions of the dependencies can be frozen. For terminal applications this might not be the case since users might end up having an environment where several applications share dependencies.

Progress is happening in the tooling to avoid using shared environments ([pipx](https://github.com/pipxproject/pipx)) but this is not true for most users.

I'll change it to `-e .` unless you tell me otherwise